ci/environment.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
   - typing-extensions
   - geoarrow-pyarrow
   - lonboard
-  - arro3-core
+  # - arro3-core  # not available on macos
```

Comment:
Oh I didn't know that the conda library wasn't building on mac ARM.

The pip installation should work anywhere